CHANGELOG

```diff
@@ -1,6 +1,12 @@
 3.0.0.pre.1
-* Adds support for recursive references
-* Removed REST api support from `Client`
+* Adds support for recursive references.
+* Removed REST api support from `Client`.
+* Added `abort` function.
+* Added `normalizer` argument to `casefold` function.
+* Deprecated `next_id` function.
```

Comment:
Can we leave a note on this line that explicitly states `next_id` is replaced by `new_id`?